dev-python/sentry-sdk/sentry-sdk-0.14.4-r1.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python3_{6..8})
+
+inherit distutils-r1
+
+DESCRIPTION="Python client for Sentry"
+HOMEPAGE="https://getsentry.com https://pypi.org/project/sentry-sdk/ https://github.com/getsentry/sentry-python"
+SRC_URI="https://github.com/getsentry/sentry-python/archive/${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
`${P}.gh.tar.gz` to avoid collision with old distfile.

Reply:
Fixed